pyproject.toml

```diff
@@ -105,6 +105,8 @@ sphinx = "6.1.3"
 sphinx_rtd_theme = "1.2.2"
 sphinxcontrib-mermaid = "0.8.1"
 myst-parser = "2.0.0"
+settings-doc = "2.0.0"
+colorama = "0.4.6" # Required on all platforms, not just win32
```

Comment:
Why is it required on all platforms? The [Colorama PyPI page](https://pypi.org/project/colorama/) clearly says "Makes ANSI escape character sequences (for producing colored terminal text and cursor positioning) work under MS Windows.".

Reply:
We could upstream an issue to them, but im guessing that dependency management does not have the correct handles to make this work based on OS.

Reply:
Python dependency management can certainly do this, `colorama` is currently only required on win32:
https://github.com/minvws/nl-kat-coordination/blob/3da1c9aa97fb84e710cd74b89e6225727cb30637/requirements.txt#L86-L88

If the `typer` package would specify its dependencies like this I think it'd work fine.

Reply:
Click and sphinx only depend on colorama on win32, but typer depends on colorama on all platforms. So I think there are two issues here that we should report upstream:
- Typer should only depend on colorama on win32
- Poetry should not add the sys_platform to the colorama dependency in requirements.txt because types doesn't limited its dependency to win32.